src/rsa/rsa.rs

```diff
@@ -29,12 +29,12 @@ use untrusted;
 mod padding;
 
 // `RSA_PKCS1_SHA1` is intentionally not exposed.
-use self::padding::{RSAPadding, RSA_PKCS1_SHA1};
+use self::padding::RSA_PKCS1_SHA1;
 pub use self::padding::{RSA_PKCS1_SHA256, RSA_PKCS1_SHA384, RSA_PKCS1_SHA512};
 
 /// Parameters for RSA verification.
 pub struct RSAParameters {
-    padding_alg: &'static RSAPadding,
+    padding_alg: &'static (padding::Verification + Sync),
```

Comment:
Let's just make `padding::Verification` `Sync`; see my suggested change in the rebase-and-tweak branch. We don't want everybody that references this stuff to have to specify `+ Sync`.